'Click',
"azure-storage-file-datalake>=12.5.0",
"azure-synapse-spark",
"azure-identity",
"py4j",
"loguru",
"pandas",
"redis",
"requests",
"pyapacheatlas",
"pyhocon",
"pandavro",
"pyyaml",
"Jinja2",
"tqdm",
"pyarrow",
"pyspark>=3.1.2",
"python-snappy",
"deltalake",
"google>=3.0.0",
"graphlib_backport",
"google-api-python-client>=2.41.0",
"azure-keyvault-secrets",
"confluent-kafka",
"databricks-cli",
"avro",
# In 1.23.0, azure-core is using ParamSpec which might cause issues in some of the databricks runtime.
# see this for more details:
# https://github.com/Azure/azure-sdk-for-python/pull/22891
# using a version lower than that to workaround this issue
"azure-core<=1.22.1",
"typing_extensions>=4.2.0"